Dockerfile

```diff
@@ -79,5 +92,10 @@ LABEL org.opencontainers.image.revision=$REVISION
 LABEL autoheal=true
 
 # Expose port and run
+
+#Transmission-RPC
 EXPOSE 9091
+# Privoxy
+EXPOSE 8118
```

Comment:
this should be optional depending on if it's enabled or not

Reply:
Agreed, but I don't believe`Dockerfile` supports conditional EXPOSE keyword ? (Correct me if I'm wrong :) )

Reply:
`EXPOSE` is more of a documentation of the image rather than an actual command. The port is determined during run time.

Ref: https://docs.docker.com/engine/reference/builder/#expose

Reply:
Will mark this as resolved then @pkishino :)